pkgs/os-specific/linux/system76-oled/default.nix

```
@@ -0,0 +1,35 @@
+{ rustPlatform, lib, fetchFromGitHub, pkg-config, makeWrapper, dbus, libX11
+, libXrandr }:
+rustPlatform.buildRustPackage rec {
```

```suggestion
{ lib
, rustPlatform
, lib
, fetchFromGitHub
, pkg-config
, makeWrapper
, dbus
, libX11
, libXrandr
}:

rustPlatform.buildRustPackage rec {
```
Makes the diff nicer, and for derivations with rather more arguments this is preferred